### PR TITLE
Observability Dashboard - Add multiselect filters and graph zoom functionality

### PR DESCRIPTION
**Pull request by Minoli**

### CODE REVIEW

File: system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/cells/MetricsGraphs.js [hunk 1]
```diff
@@ -68,14 +72,21 @@ const styles = {
...
     barChart: {
         marginTop: 40,
         marginBottom: 40
+    },
+    info: {
+        color: "#999",
+        marginTop: 8,
+        marginRight: 27,
+        fontSize: 18
+
```
Comment (dakshika) [inline system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/cells/MetricsGraphs.js]: unwanted new line

File: system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/systemMetrics/common/Metrics.js [hunk 2]
```diff
@@ -53,6 +57,13 @@ const styles = {
     },
     toolTipHead: {
         fontWeight: 600
+    },
+    info: {
+        color: "#999",
+        marginTop: 8,
+        marginRight: 27,
+        fontSize: 18
+
```
Comment (dakshika) [inline system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/systemMetrics/common/Metrics.js]: unwanted new line

File: system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/cells/MetricsGraphs.js (hunk 1 above)
Comment (Minoli) [reply system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/cells/MetricsGraphs.js]: Removed in https://github.com/wso2-cellery/mesh-observability/blob/master/components/global/portal/io.cellery.observability.ui/src/components/cells/metricsGraphs/index.js

File: system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/systemMetrics/common/Metrics.js (hunk 2 above)
Comment (Minoli) [reply system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/systemMetrics/common/Metrics.js]: Removed in developement repo https://github.com/wso2-cellery/mesh-observability/blob/master/components/global/portal/io.cellery.observability.ui/src/components/cells/metricsGraphs/index.js

**Pull request merged**